adapter/src/main/java/org/jvalue/ods/adapterservice/adapter/api/rest/v1/Mappings.java

```diff
@@ -1,9 +1,22 @@
 package org.jvalue.ods.adapterservice.adapter.api.rest.v1;
 
+import java.io.IOException;
+import java.util.Map;
+
+import org.jvalue.ods.adapterservice.adapter.model.exceptions.*;
+import org.springframework.http.HttpStatus;
+
 public class Mappings {
   public static final String IMPORT_PATH = "/preview";
   public static final String RAW_IMPORT_PATH = "/preview/raw";
   public static final String FORMAT_PATH = "/formats";
   public static final String PROTOCOL_PATH = "/protocols";
   public static final String VERSION_PATH = "/version";
+
+  public static final Map<Class, HttpStatus> ERROR_MAPPING = Map.of(
```

Comment:
I wonder if a status code annotation directly at the exception is better?
```java
@ResponseStatus(code = HttpStatus.BAD_REQUEST)
class CustomException extends RuntimeException {}
```
Probably you had some thoughts on that, do you mind sharing them? :)

Reply:
> I wonder if a status code annotation directly at the exception is better?
> 
> Probably you had some thoughts on that, do you mind sharing them? :)

I honestly had no clue about the way you are "supposed" to do it in Spring. Gonna have to take a look at it in detail. The solution I went for was the first one that came to mind to avoid repetition in every endpoint method.